php_1/CustomerControllerClass.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+require_once('db/PdoConnectionClass.php');
+
+class CustomerControllerClass
+{
+    protected string $tableName;
+
+    public function __construct(string $tableName)
+    {
+        $this->tableName = $tableName;
+    }
+
+    public function getCustomers(): array
+    {
+        $conn = new PdoConnectionClass();  
```

Comment:
this is aduplicare statement used in every class function. To avoid creating multiple database connections for the same customerControllerClass object, move this code inside the class constructor.